homeassistant/components/tado/__init__.py

```diff
@@ -203,18 +207,35 @@ def update_mobile_devices(self) -> None:
             _LOGGER.error("Unable to connect to Tado while updating mobile devices")
             return
 
+        if len(mobile_devices) == 0:
+            _LOGGER.warning(
+                "No linked mobile devices found for home ID %s", self.home_id
+            )
+            return
+
+        if "errors" in mobile_devices and len(mobile_devices["errors"]) > 0:
```

Comment:
Why does this container have an `errors` item? I'd expect the client call to raise an exception if there's an error. Returning errors like this is really not good practice. Can we update the library instead?

Reply:
The library update can happen in a follow up. Ok to handle the errors here for now.